plasmapy/physics/parameters.py

```diff
@@ -713,32 +716,42 @@ def collision_rate_ion_ion(T_i, n_i, ion_particle,
     <Quantity 95918.76240877 1 / s>
 
     """
-    from plasmapy.physics.transport.collisions import Coulomb_logarithm
+    from plasmapy.physics.transport.collisions import (Coulomb_logarithm,
+                                                       collision_frequency)
     T_i = T_i.to(u.K, equivalencies=u.temperature_energy())
     m_i = atomic.ion_mass(ion_particle)
```

Comment:
There may be a conflict here with #340, but I think that can be resolved by switching `ion_mass` to `particle_mass` here.  It would probably make sense for this PR to be merged first, and then I can fix the conflicts in #340 after that.